controllers/managedocs_controller.go

```diff
@@ -592,6 +615,18 @@ func (r *ManagedOCSReconciler) reconcileDMSPrometheusRule() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileOCSInitialization() error {
+	r.Log.Info("Reconciling OCSInitialization")
+
+	if err := r.get(r.ocsInitialization); err == nil {
+		r.ocsInitialization.Spec.EnableCephTools = true
+		if err = r.Client.Update(r.ctx, r.ocsInitialization); err != nil {
+			return err
+		}
+	}
```

Comment:
A couple of points:
1. I think we do want to fail the reconcile if the error is no `errors.IsNotFound` because it means that the deployer has a genuine issue retrieving the object that we would like to identify and fix (for example RBAC issues)
2. I will consider returning the underlaying error (of a generic Get or Create operation) without encapsulating it with a context a bad practice. let's add a `fmt.ErrorF` with a more meaningful error" *(you can still attach the original error message to the new one)
3. If we don't find the resource (IsNotFound) I think it will be helpful to document that in the log as a Warning or an Info